test/library/packages/Search/correctness.chpl

```diff
@@ -2,7 +2,7 @@
  *  Check correctness of search functions
  */
 
-use Search;
+use Search; use Sort only relativeComparator;
```

Comment:
Since Search re-exports comparators, should it also re-export their interfaces?  (We don't have to decide that right now, but I'd probably make an issue about the question for when we stabilize Search)

Reply:
https://github.com/chapel-lang/chapel/issues/25849